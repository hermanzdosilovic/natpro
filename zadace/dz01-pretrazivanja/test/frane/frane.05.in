1000
4331 3775 985 3423 1280 1906 1677 1523 1792 1626 257 3239 4588 3028 2243 616 2420 3052 4243 2046 1744 2500 1535 1694 4426 992 159 4802 2309 3193 1876 1839 2515 1313 4867 2645 929 4134 3446 2498 1326 1299 4025 3563 2805 1316 4952 4608 4223 619 3169 599 83 3855 767 4610 4985 3114 730 3128 183 3724 1744 3077 3335 4502 70 666 2926 3773 1108 4531 4490 4487 4313 4568 2911 4603 2339 1794 2632 1922 1494 2088 1673 2422 209 151 995 999 3945 1239 4124 3984 1691 1118 3530 2530 3533 4289 2104 3938 3961 2554 3652 1519 2888 336 4313 370 4938 2090 3161 4626 4179 4097 2869 3000 4272 1514 3959 2736 2640 3159 4463 743 2255 3191 3710 2631 1201 209 1329 3508 1527 728 914 1692 921 3918 2459 3744 4939 498 2864 3283 4007 731 1203 4731 4566 414 3015 3458 4686 1777 3002 4770 1285 83 1069 4282 1360 2710 4870 4156 1016 3712 2387 3344 284 4600 4962 1338 15 3194 2128 158 3060 2614 3548 4445 3368 4064 4213 2049 3517 102 1351 112 2754 2997 679 1591 544 4044 3861 4758 2641 4887 4832 3651 3286 3326 2874 3319 4585 1518 2079 566 2645 1405 134 535 3024 49 1307 2678 3900 3356 3137 3397 3067 467 1344 1836 2812 1482 2756 3630 1950 4126 4532 1783 487 3545 4320 470 986 902 2574 280 1518 3648 3780 3754 4954 54 1982 1277 1322 2101 552 4323 3567 3993 3805 2523 2995 4618 3191 818 4480 921 3645 1771 1400 1239 227 804 816 4706 1551 2912 2303 3170 3821 4018 4848 4080 2405 584 2324 80 3468 3649 854 3777 4982 3796 2995 726 3641 1657 165 3600 3463 4287 1193 3368 3604 3604 2785 2040 2787 147 2209 1464 263 1725 4333 4032 1127 2673 2666 275 1481 2135 1519 4349 524 3440 3495 495 1513 4043 2669 95 4701 4281 3573 3847 3674 4504 4813 4959 2680 160 1012 4146 1509 848 1859 3639 692 783 4290 1062 2172 2065 3240 2093 3129 2449 1108 1415 174 868 1903 1292 2523 944 2348 3922 3947 4385 3717 4479 4739 1044 4071 4860 1846 1848 3333 3322 1112 3808 1487 4718 2964 158 3681 2832 2995 1331 3210 3282 4726 152 680 2161 1313 4330 3665 1830 4668 2092 3246 3379 3665 1390 1955 2564 2474 3422 217 813 1159 2944 2812 2015 3575 1464 4474 1374 4119 3034 1847 126 3211 3612 4451 4549 217 3631 2440 4400 4425 4278 4443 1428 2482 3359 4036 1923 4338 1887 2380 4845 4443 3708 2022 3333 3365 3474 3504 4272 3056 4909 4104 542 1309 3944 2946 4635 3172 2097 2008 1078 2193 3322 4099 3162 751 260 4672 4256 1318 4244 125 1211 3976 2045 1281 3445 710 624 4199 607 3258 4120 2211 1683 4987 1717 1684 584 3595 533 401 3830 2800 2239 2495 1521 2690 1048 2764 1676 1888 1665 4025 2060 2650 3793 862 3817 2171 2971 2011 4986 540 1119 4124 4374 4523 3230 697 4045 25 3116 1730 218 1819 1427 3917 3615 1923 4451 395 2181 1952 1505 4714 2668 3093 188 419 2294 3433 328 3314 2197 2366 921 3354 941 3270 2397 2982 3758 4978 4045 1142 1186 4743 3002 3297 2493 2226 1589 2633 1792 2975 4440 4951 1501 2646 3560 3595 4171 753 1066 933 3483 4597 4356 3920 3639 3672 1191 1088 3705 4265 2506 4766 3000 3860 69 2403 370 3245 3222 234 2935 2834 2225 4424 1362 1407 972 2892 2941 291 2865 3912 481 898 8 2751 1885 4103 4625 4575 2784 4924 469 383 4153 356 541 384 430 3724 456 1325 3892 1429 4350 618 1725 370 2760 1859 2798 3441 2955 1099 992 946 3462 1655 4340 3870 390 3994 4967 1114 1073 4247 307 4642 262 4213 3085 4576 2900 2227 4744 936 4774 2355 2450 78 1183 4312 4189 4398 4953 2864 3160 125 666 2246 2724 3757 829 3772 3813 372 4705 1083 1269 2910 1011 1449 1592 3278 2459 1678 3041 2537 4887 4212 1759 4013 320 1781 943 742 1746 4483 4197 4463 442 848 4029 807 3679 1684 2366 3174 3947 4497 3934 3332 4307 1374 216 4801 418 1285 509 813 2486 1938 2956 3147 448 1767 1127 4503 1535 833 1606 3235 3914 1844 1634 2164 2412 4434 4306 1825 4941 4034 1590 4847 736 2254 4933 2442 4540 426 4277 23 2929 4069 2783 2368 1226 3300 616 3547 2979 75 2098 1221 1728 4957 2289 288 3404 1068 4971 4087 2958 1680 1171 4942 1315 1222 2240 4854 4426 863 3970 2589 2359 4701 1182 1217 391 2287 1233 2471 4591 3906 2554 613 4475 41 279 1980 758 3791 3402 254 2901 4959 2915 4681 1056 611 4554 276 4601 516 3987 3593 2364 4161 1407 945 3780 3861 1264 1679 3810 676 1717 4629 3064 4129 2165 3714 224 1672 2640 400 2182 1022 3929 701 190 4821 2781 1322 1685 1293 4996 2128 4945 47 1445 868 4284 2310 4039 4726 4394 3629 2381 3119 2827 4126 931 1304 4967 1991 2399 1845 1249 4087 9 1148 578 685 4570 1358 2488 4980 2097 4137 359 158 4574 612 3734 2931 4013 1175 4934 1279 2268 2723 1212 2430 4569 3158 4747 4175 57 2890 1946 2365 1921 2880 3024 1323 660 4181 686 4537 859 1834 2652 41 3156 4493 1780 2265 2106 2693 4710 3234 3605 2079 63 2789 814 1520 4961 3790 3356 3491 1001 2297 2939 4718 4049 2955 3792 3932 2947 2547 2646 979 964 472 2511 2267 4744 3121 3310 3700 1512 3969 1911 1640 2576 3158 1014 3590 3231 4930 1691 2464 4510 3965 3555 2869 1154 873 4966 4619 4046 790 3224 2084 2714 1630 2158 431 4521 393 4692 1083 3376 4035 1237 430 244 3507 2065 1160 503 995 2074
