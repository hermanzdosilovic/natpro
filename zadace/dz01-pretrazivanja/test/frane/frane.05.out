2104092999.100
